tasks/main.yml

```diff
@@ -127,3 +129,12 @@
     name: docker.io
     state: started
   when: docker_pkg_name == 'docker.io'
+
+  # ATTENTION: this task can potentially create new users!
```

Comment:
we could provide a "safer" behaviour with a (maybe non-idempotent)  ̀command` task running some  ̀usermod` call, but I personnally prefer to rely on Ansible core modules, with this little drawback.